packages/sources/gsr/src/config/index.ts

```diff
@@ -27,4 +27,30 @@ export const config = new AdapterConfig({
     required: true,
     sensitive: true,
   },
+  LWBA_API_ENDPOINT: {
+    type: 'string',
+    description: 'The HTTP API endpoint to use to retrieve tokens for LWBA',
+    default: 'https://oracle.pre-prod.gsr.io/v1',
+  },
+  LWBA_WS_API_ENDPOINT: {
+    type: 'string',
+    description: 'The WS API endpoint to use for LWBA data',
+    default: 'wss://oracle.pre-prod.gsr.io/oracle',
+  },
+  LWBA_WS_USER_ID: {
+    type: 'string',
+    description: 'The user ID used to authenticate to the LWBA_API_ENDPOINT',
+    required: true,
+  },
+  LWBA_WS_PUBLIC_KEY: {
+    type: 'string',
+    description: 'The public key used to authenticate to the LWBA_API_ENDPOINT',
+    required: true,
+  },
+  LWBA_WS_PRIVATE_KEY: {
+    type: 'string',
+    description: 'The private key used to authenticate to the LWBA_API_ENDPOINT',
+    required: true,
+    sensitive: true,
+  },
```

Comment:
Ideally we use the same credentials as the other endpoint(s), so we could delete these. Again if the DP can't work with us on it, this is the best we can do.